README.md

```diff
@@ -0,0 +1,214 @@
+<a name="#readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [School-library](#School-library) <a name="about-project"></a>
+
+The School-library project is an application that allows an efficient use of libraries especially in school areas. The administrators can perform the tasks such as add, delete and update information about books and publications as well as lending books to students or other registerd users as the teachers and administration employees. Users, students, teachers and administration employees can borrow books through the application. The School-library app was created with Ruby.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The following technologies have been used for the project development.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Rails.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The key features of the application are:
+
+- **[Create and delete books]()**
+- **[List books]()**
+- **[Select book]()**
+- **[Filter books]()**
+- **[Create and delete users]()**
+- **[List users]()**
+- **[Select user]()**
+- **[Filter users]()**
```

Comment:
[OPTIONAL]
- [ ] I kindly advice that you do not make these key features as links and you can do this by removing the parentheses`()` in front of the texts :+1: :sparkles: